flags/src/main/java/org/secondbase/flags/Flags.java

```diff
@@ -564,11 +564,11 @@ protected SecretPath getS3Path(final Object value) {
             return null;
```

Comment:
It's not really an exception if it's the most common case. How about an Optional<SecretPath> or something like that?